src/app/containers/CpsAssetMediaPlayer/index.jsx

```diff
@@ -78,11 +83,15 @@ CpsAssetMediaPlayer.propTypes = {
   assetUri: string.isRequired,
   isLegacyMedia: bool,
   showLoadingImage: bool,
+  bottomPadding: bool,
+  showCaption: bool,
 };
 CpsAssetMediaPlayer.defaultProps = {
   ...emptyBlockArrayDefaultProps,
   isLegacyMedia: false,
   showLoadingImage: false,
+  bottomPadding: true,
+  showCaption: true,
```

Comment:
Is there a reason to expose `bottomPadding` as a prop, can we just apply the padding when `showCaption` is true?

Reply:
the bottom padding should actually be visible on all players, this override was just included as MAP's did not want the bottom padding.

So the use of showCaption and bottomPadding are not exclusive